src/Symfony/Component/Form/ChoiceList/Factory/DefaultChoiceListFactory.php

```diff
@@ -245,6 +253,9 @@ private static function addChoiceViewsGroupedByCallable($groupBy, $choice, $valu
                 $preferredViews[$groupLabel] = new ChoiceGroupView($groupLabel);
                 $otherViews[$groupLabel] = new ChoiceGroupView($groupLabel);
             }
+            if (!isset($preferredViewsOrder[$groupLabel])) {
```

Comment:
Minor comment but I would rather remove this if and use `$preferredViewsOrder[$groupLabel] ?? []` in the method call below.

Reply:
It is passed by reference, I do not think this would work.

https://github.com/symfony/symfony/blob/8afaaff1849d8d530524e908577e2509e279c089/src/Symfony/Component/Form/ChoiceList/Factory/DefaultChoiceListFactory.php#L137